lib/AST/ASTImporter.cpp

```diff
@@ -6334,6 +6308,26 @@ Expr *ASTNodeImporter::VisitSubstNonTypeTemplateParmExpr(
         Replacement);
 }
 
+Expr *ASTNodeImporter::VisitTypeTraitExpr(TypeTraitExpr *E) {
```

Comment:
why the function has to be moved?
If this version is going to be upstreamed and you think it is better to move it as well, then I am fine with that. Just see no reason for the line change.